include/odp/api/spec/classification.h

```diff
@@ -317,11 +315,37 @@ typedef enum {
 	ODP_COS_FHDR_USER	/**< Application-specific header field(s) */
 } odp_cos_hdr_flow_fields_t;
 
+/**
+ * Enumeration of actions for CoS.
+ */
+typedef enum {
+	/**
+	 * Accept packet
+	 *
+	 * Packets that arrive in the CoS are accepted and their processing
+	 * continues towards other chained CoSes or the destination queue.
+	 */
+	ODP_COS_ACTION_ACCEPT,
+
+	/**
+	 * Drop packet
+	 *
+	 * Packets that arrive in the CoS are dropped. Packets are freed into
+	 * their originating pool.
+	 */
+	ODP_COS_ACTION_DROP,
+} odp_cos_action_t;
+
 /**
  * Class of service parameters
  * Used to communicate class of service creation options
  */
 typedef struct odp_cls_cos_param {
+	/** Action to take. When action is ODP_COS_ACTION_DROP, all the other
+	 * parameters are ignored. Default is ODP_COS_ACTION_ACCEPT.
```

Comment:
Another naming option for accept CoS could be ACTION_ENQUEUE. It would allow us to add other accept type actions later.